developers/index.md

```diff
@@ -43,82 +43,35 @@ General [coding guidelines](development/guidelines.html) apply to all types of a
-1. In Eclipse Package Explorer: look for `app.bndrun` in the `org.openhab.demo.app` project.
-   Double click to open `app.bndrun` (takes a few seconds)
-1. Under Browse Repos search for your binding (`astro`) and add it to the "Run Requirements" list using drag&drop from the "Browse Repos" list:
-    ![Bndtools](images/ide_setup_eclipse_bndtools.png)
-1. Click "Resolve": a window with the list of resolved bundles will be shown. 
-    Click Finish and save the file.
-    
-    Now the IDE is ready to start openHAB with a minimum set of the openHAB core bindings, PaperUI and the selected binding 
-1. Start openHAB from the IDE clicking "Run OSGi" or "Debug OSGi".
-   You can check openHAB is running going with your browser to: http://localhost:8080/paperui/
-1. Check the chosen binding is active in PaperUI -> Configuration -> Bindings
-
-## Develop a NEW binding with the IDE 
-
-To start developing a new binding follow these steps:
-
-1. Close Eclipse
-1. From the command line in `openhab2-addons/bundles` directory run:
-
-    `$ ./create_openhab_binding_skeleton.sh  MyNewBinding <Author> <GitHubUsername>`
-    
-    to create a skeleton of a new binding `mynewbinding`.
-    Accept with `Y` the skeleton configuration.
-1. Edit file `openhab2-addons/bom/openhab-addons/pom.xml`
-and add your new binding to the list (adapt the artifactid to your binding name!):
-    ```
-    <dependency>
-        <groupId>org.openhab.addons.bundles</groupId>
-        <artifactId>org.openhab.binding.mynewbinding</artifactId>
-        <version>${project.version}</version>
-    </dependency>
-    ```
-1. From the `openhab2-addons` directory run: ` mvn -DskipChecks -DskipTests clean install` to rebuild the list of bindings in the BOM (Bill Of Material) and make your new binding visible from the demo.app project
-1. Now open Eclipse and follow from previous step 6. to import and run/debug your new binding in the Eclipse IDE
-
-Now you can start developing your new binding. 
-
-In case the new binding has dependencies to other bundles/libraries see [Adding Dependencies](buildsystem.html#adding-dependencies) for more information.
+1. [Git](https://git-scm.com/downloads) For retrieving our source code and push changes back. On Windows: Must be available in %PATH%
+1. [Maven 3.x](https://maven.apache.org/download.cgi) Our buildsystem tool. On Windows: Must be available in %PATH%
+1. Java JDK 8, for example from Oracle [Oracle JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html). On Windows: %JAVA% must be set.
+
+You can use any IDE that is suitable for OSGi/Java development.
+We have prepared some step-by-step guides for the following IDEs:
+
+<table style="width:100%">
+<tr>
+<td style="width:30%">
+
+[![Visual Studio Code](ide/images/vscode.jpg)](ide/vscode.html)
+
+</td>
+<td style="width:30%">
+
+[![Eclipse IDE](ide/images/vscode.jpg)](ide/eclipse.html)
```

Comment:
Try adding `./` before, this might work (not 100% sure).
(also the image is wrong here)
`[![Eclipse IDE](ide/images/eclipse.jpg)](ide/eclipse.html)`

Ref: https://vuepress.vuejs.org/guide/assets.html#relative-urls

Reply:
> (also the image is wrong here)

Wrong means it cant be used as a url text?

Reply:
no it's _vscode.jpg_ but should be _eclipse.jpg_ :)

Reply:
Ah. Seems i have to clean my glasses....